client/src/reducers/config.ts

```diff
@@ -1,12 +1,24 @@
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-const Config = (
-  state = {
+import { AnyAction } from "redux";
+import type { Config } from "../globals";
+
+interface ConfigState {
+  features: Config["features"] | null;
+  parameters: Config["parameters"] | null;
+  displayNames: Config["displayNames"] | null;
+  loading?: boolean;
+  error?: Error | string | null;
```

Comment:
This seems like the result of a downstream bug, shouldn't we restrain ourselves to either Error or string, not both?